index.html

```diff
@@ -0,0 +1,177 @@
+          </p>
+          <div class="date">2022.03.11(MON) ~ (FRI)</div>
+          <p class="venue">@DOMINION CITY Ajah-Lagos,Nigeria.</p>
+        </div>
+
+        <!-- Programs -->
+        <article class="home-section_program">
+          <div class="program_h2">
+            <h2 class="main_program">Main Program</h2>
+            <hr class="program_hr" />
+          </div>
+          <div class="program">
+            <img src="./img/lecture.png" alt="lecture-icon" />
+            <h3>Lecture</h3>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Iusto
+              modi velit facilis sit fugit eveniet
+            </p>
+          </div>
+          <div class="program">
+            <img src="./img/workshop.png" alt="" />
+            <h3>Workshop</h3>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Eaque
+              doloribus ipsa qui incidunt facere.
+            </p>
+          </div>
+          <div class="program">
+            <img src="./img/lecture.png" alt="">
+            <h3>HeartCare</h3>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Eligendi
+              veritatis ea aliquid eos inventore?
+            </p>
+          </div>
+          <div class="program">
+            <img src="./img/forum.png" alt="health-image" />
+            <h3>Forum</h3>
+            <p>
+              Lorem ipsum, dolor sit amet consectetur adipisicing elit.
+              Laudantium corporis porro dolore mollitia autem.
+            </p>
+          </div>
+          <div class="program">
+            <img src="./img/ignite.png" alt="health-image" />
+            <h3>Ignite</h3>
+            <p>
+              Lorem, ipsum dolor sit amet consectetur adipisicing elit. Eius
+              neque impedit expedita est repudiandae.
+            </p>
+          </div>
+          <button type="button" class="program-btn">Join Fitnesss Family-2022</button>
+          <span class="see_program">
+            <a href="#">see the whole program</a>
+          </span>
+        </article>
+
+        <!-- Speakers -->
+        <article class="speakers">
+          <div class="speaker_h2">
+            <h2>Featured Speakers</h2>
+            <hr class="program_hr" />
+          </div>
+
+          <!-- Speakers Details -->
+          <div class="speaker_container"></div>
+        </article>
+      </section>
+
+      <footer>
+        <div class="footer_bg">
+        <div class="program_h2">
+          <h2 class="main_program">Partner</h2>
+          <hr class="program_hr"/>
+        </div>
+
+        <ul class="footer_list">
+          <li><img src="./img/dc.jpeg" alt="" /></li>
+          <li><img src="./img/who.png" alt="" /></li>
+          <li><img src="./img/dinma.jpg" alt="" /></li>
+          <li><img src="./img/food-logo.jpg" alt="" /></li>
+          <li><img src="./img/nike.jpg" alt="" /></li>
+        </ul>
```

Comment:
For the moment, the partner's section takes the whole height of my screen, mostly because of the size of one logo.

Your design:
![image](https://user-images.githubusercontent.com/83596135/163587165-50f213cb-2520-46d6-8dd3-67befe057863.png)

- [ ] Kindly, reduce in size of this section, to match the given design;

Given design:
![image](https://user-images.githubusercontent.com/83596135/163587274-27037be7-1168-47ef-98a3-9a82a18ae394.png)